### PR TITLE
Get android-netguard from maven instead of jitpack

### PR DESCRIPTION
**Pull request by aitorvs**
Task/Issue URL: https://app.asana.com/0/488551667048375/1203722248572744/f

### Description
Start consuming the android-netguard library from maven instead of Jitpack

### Steps to test this PR
* App compiles
* Smoke tests on AppTP

### CODE REVIEW

**Comment by aitorvs** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #2729** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/2729" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/duckduckgo/Android/2729?utm_source=stack-comment).

File: vpn-network/vpn-network-impl/build.gradle [hunk 1]
```diff
@@ -33,7 +33,7 @@ dependencies {
 
     implementation Kotlin.stdlib.jdk7
     implementation "com.squareup.logcat:logcat:_"
-    implementation "com.github.duckduckgo:netguard:1.0.0"
+    implementation "com.duckduckgo.netguard:netguard-android:1.1.0"
```
Comment (karlenDimla) [inline vpn-network/vpn-network-impl/build.gradle:36]: Why not set this to _ and let the version handled by versions.properties?

**Review by karlenDimla — APPROVED** [review]
One nit but it works as expected.

File: vpn-network/vpn-network-impl/build.gradle (hunk 1 above)
Comment (aitorvs) [reply vpn-network/vpn-network-impl/build.gradle:36]: oh, absolutely, forgot about that :D Doing it now

**Pull request merged**